pkg/webhooks/reconciller.go

```diff
@@ -122,6 +124,7 @@ func (reconciler *ValidatingWebhookReconciler) Reconcile(ctx context.Context, cl
 	}
 
 	_, err := controllerutil.CreateOrUpdate(ctx, client, cr, func() error {
+		ownerutil.EnsureOwner(cr, owner)
```

Comment:
Same as for MutatingWebhookConfiguration above, don't set owner reference.
Removal of cluster scoped objects should be handled via finalizers, but we can solve this in a follow up task.
Ping me in GChat if you need more details. :)